libs/core/combobox/combobox.component.html

```diff
@@ -133,11 +133,11 @@
     >
         <ng-content></ng-content>
         @if (groupFn) {
-            @for (group of displayedValues | listGroupPipe: groupFn; track group) {
+            @for (group of displayedValues | listGroupPipe: groupFn; track group.key) {
                 <li role="group" fd-list-group-header [tabindex]="0">
                     <span fd-list-title>{{ group.key }}</span>
                 </li>
-                @for (term of group.value; track term) {
+                @for (term of group.value; track term.name) {
```

Comment:
This only works with the given stackblitz example because the items in the example have a `name` property. If the example code were instead
```
export type ComboboxItem = { testName: string; type: string };

    dropdownValues: ComboboxItem[] = [
        { testName: 'Apple', type: 'Fruits' },
        { testName: 'Banana', type: 'Fruits' },
        { testName: 'Pineapple', type: 'Fruits' }
```

it will still throw the warning

Reply:
There's no need to add tracking to this, as you're right—we don’t know what type the combobox can accept. We could use indexes, but I’m not sure if that’s the best approach. Without the tracking function, it works as expected, and we’re not seeing any warning messages. @mikerodonnell89